### PR TITLE
Add get credit by type

### PR DESCRIPTION
**Pull request by jbirchevans**
Added a function to get the credits for a service by the name

### CODE REVIEW

**Comment by tylerwoonton** [comment]
@jbirchevans Might want to write a test for this new functionality too, but what you have at the min is fine.

File: src/Account/CreditClient.php [hunk 1]
```diff
@@ -36,6 +36,20 @@ public function getAll()
         }, $body->data);
     }
 
+    /**
+     * Get credits for an individual service
+     * @param $reference
+     * @return Credit
+     * @throws \GuzzleHttp\Exception\GuzzleException
+     */
+    public function findByReference($reference)
```
Comment (phily245) [inline src/Account/CreditClient.php]: Should this be `getByReference()` to match our naming conventions?

**Comment by phily245** [comment]
Got a question on the MR about standardized naming

File: src/Account/CreditClient.php (hunk 1 above)
Comment (jbirchevans) [reply src/Account/CreditClient.php]: Good point, I will make the changes now.

Comment (jbirchevans) [reply src/Account/CreditClient.php]: @phily245 this has now been updated

**Pull request merged**